test/src/v1/types/journey.ts

```diff
@@ -0,0 +1,18 @@
+export type TripsSimplifiedResponseType = Array<{
+  expectedStartTime: string;
+  expectedEndTime: string;
+  distance: number;
+  legs: Array<{
+    fromPlace: {
+      name: string;
+    };
+    toPlace: {
+      name: string;
+    };
+    serviceJourney: {
+      id: string;
+    };
+  }>;
+}>;
+
+export type TripsSingleSimplifiedResponseType = TripsSimplifiedResponseType[0];
```

Comment:
Would it not make more sense to make the `TripsSingleSimplifiedResponseType` be the object, then `TripsSimplifiedResponseType = TripsSingleSimplifiedResponseType[]`?

Reply:
Nice catch! I'll change.